packages/ui/src/components/TextField/STextField.vue

```diff
@@ -159,6 +160,11 @@ const counterText = computed<string | null>(() => {
   return limit === null ? String(currentCount) : `${currentCount}/${limit}`
 })
 
+const attrs = useAttrs()
+const rootClass = computed(() => attrs.class)
+const rootStyle = computed(() => attrs.style) as StyleValue
```

Comment:
Not `computed(() => ...) as StyleValue`, but `computed(() => ... as StyleValue)`. It is `ComputedRef<StyleValue>`.